### ISSUE TITLE
Section suspension does not assign Vet Ban for Veteran Raids

### ISSUE THREAD

**Issue by Deaththth** [ewang2002/ToogaBooga#101]
There is no way to configure a role for it at the moment either.

**Comment by ewang2002** [comment]
A role is unnecessary. This is internally logged in the database and you can check if the person is section suspended by using an appropriate command. I also do not plan on adding a role that designates this.

What do you have in mind?

**Comment by Deaththth** [comment]
Wait, so does it use channel permissions to only ban the player from that section or from verification for that section?

**Comment by ewang2002** [comment]
All the bot needs to do is remove the section member role from the person and then prevent that person from verifying into said section.

Editing channel permissions is not feasible due to the possibility that we need to make hundreds of overrides.

**Comment by ewang2002** [comment]
If a role is preferred, I can add it at a later time.

**Comment by Deaththth** [comment]
I agree with your approach, we need to handle vet verification through the bot next then for it to work.
